### PR TITLE
fuse_ll: fix compile error when libfuse version is larger than 3.0

### PR DESCRIPTION
**Pull request by lxbsz**
Fixes: https://tracker.ceph.com/issues/56517
Signed-off-by: Xiubo Li <xiubli@redhat.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: src/client/fuse_ll.cc [hunk 1]
```diff
@@ -1370,13 +1370,13 @@ const static struct fuse_lowlevel_ops fuse_ll_oper = {
...
 CephFuse::Handle::Handle(Client *c, int fd) :
   fd_on_success(fd),
-  client(c),
-  se(NULL),
+  client(c)
+{
 #if FUSE_VERSION < FUSE_MAKE_VERSION(3, 0)
-  ch(NULL),
-  mountpoint(NULL)
+  ch = nullptr;
```
Comment (tchaikov) [inline src/client/fuse_ll.cc:1376]: why not just initialize `ch` and `mountpoint` inlined, i.e. to initialize them at line 178 and line 179.

Comment (lxbsz) [reply src/client/fuse_ll.cc:1376]: Sounds good. Will fix it.

**Review by tchaikov — APPROVED** [review]

**Comment by lxbsz** [comment]
> I confirm that this fixes the issue I saw. Thanks @lxbsz

Cool, thanks!

**Comment by vshankar** [comment]
ceph test api

**Comment by vshankar** [comment]
ceph test windows

**Comment by vshankar** [comment]
jenkins test windows

**Review by vshankar — APPROVED** [review]

**Pull request merged**